src/display/editor/toolbar.js

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2023 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { noContextMenu } from "../display_utils.js";
+
+class EditorToolbar {
+  #div = null;
+
+  #editor;
+
+  #icons = null;
```

Comment:
It feels slightly strange to, in the `render`-method below, append *buttons* to an element called "icons".
Should we name this property e.g. `#buttons` (or possibly `#container`) instead, and similar for the CSS rules?